projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -48,6 +48,9 @@ function jetpack_setup_blogging_prompt_response( $post_id ) {
 	if ( $prompt ) {
 		update_post_meta( $post_id, '_jetpack_blogging_prompt_key', $prompt_id );
 		wp_add_post_tags( $post_id, array( 'dailyprompt', "dailyprompt-$prompt_id" ) );
+		if ( array_key_exists( 'bloganuary_id', $prompt ) ) {
```

Comment:
`jetpack_get_blogging_prompt_by_id` returns a stdClass object from what I can tell. I think this should be something like;

```suggestion
		if ( property_exists( $prompt, 'bloganuary_id' ) ) {
```